django/econsensus/publicweb/static/css/styles.css

```diff
@@ -111,6 +111,16 @@ a.button {
 	top:0.2em;
 }
 
+a.toggle {
```

Comment:
my understanding from https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Writing_efficient_CSS is that it's not good practice to qualify a class rules with a tag name. Is there a specific reason why `.toggle` wouldn't do here?